drracket/drracket/private/unit.rkt

```diff
@@ -2355,6 +2355,9 @@
           (toggle-show/hide-definitions)
           (update-shown)))
       (define/public (ensure-rep-shown rep)
+        (unless (is-a? rep drracket:rep:text<%>)
+           (error 'ensure-rep-shown
```

Comment:
The documentation of `error` states:

> When possible, use functions such as `raise-argument-error`, instead, which construct messages that follow Racket’s error message conventions.

So let's use `raise-argument-error` instead.

Reply:
Oh, and thanks to @rfindler for catching this!